test/hydration-test.js

```diff
@@ -3,176 +3,104 @@ import Hydration from '../src/Hydration';
-      id: 4,
+    user2 = new User({
+      id: 2,
       name: "Rainbow Dash",
       address: "1237 Equestria Street, Denver CO 80301-1697",
       email: "rainbowD1@hotmail.com",
       strideLength: 3.8,
       dailyStepGoal: 7000,
       friends: [1, 2, 3]
     });
-    const users = [user3, user4];
-    const userRepo = new UserRepo(users);
-    // console.log(hydration.calculateFirstWeekOunces(userRepo, 4));
-    expect(hydration.calculateFirstWeekOunces(userRepo, 4)[0]).to.eql('2019/09/20: 40');
-    expect(hydration.calculateFirstWeekOunces(userRepo, 4)[6]).to.eql('2019/04/15: 36');
+
+    users = [user1, user2];
+    userRepo = new UserRepo(users);
+    hydration = new Hydration(hydrationData);
   });
 
-  it('should find sleep quality by day for that days week', function() {
-    const user3 = new User({
-      id: 3,
-      name: "The Rock",
-      address: "1236 Awesome Street, Denver CO 80301-1697",
-      email: "therock@hotmail.com",
-      strideLength: 10,
-      dailyStepGoal: 60000,
-      friends: [1, 2, 4]
-    });
+  it('should be a function', () => {
+    expect(Hydration).to.be.a('function');
+    expect(hydration).to.be.an.instanceof(Hydration);
+  });
 
-    const user4 = new User({
-      id: 4,
-      name: "Rainbow Dash",
-      address: "1237 Equestria Street, Denver CO 80301-1697",
-      email: "rainbowD1@hotmail.com",
-      strideLength: 3.8,
-      dailyStepGoal: 7000,
-      friends: [1, 2, 3]
-    });
-    const users = [user3, user4];
-    const userRepo = new UserRepo(users);
-    console.log("HELOOO", hydration.calculateRandomWeekOunces('2018/02/01', 4, userRepo));
-    expect(hydration.calculateRandomWeekOunces('2019/09/18', 4, userRepo)[0]).to.eql('2019/09/18: 40');
-    // expect(hydration.calculateRandomWeekOunces('2018/02/01', 4, userRepo)[6]).to.eql('2019/09/16: 30');
-    //this is failing because it doesn't exist, need a failure case
-  })
-  //day of hydration should not include user 2 or user 1 on August 22
-  //week of hydration should not include user 4 not during the week
+  it('should take in a list of data', () => {
+    expect(hydration.hydrationData[0].userID).to.equal(1);
+    expect(hydration.hydrationData[2].numOunces).to.equal(30);
+    expect(hydration.hydrationData[4].date).to.equal("2018/10/23");
+  });
+
+  it('should find the average water intake per day for a user', () => {
+    expect(hydration.calculateAverageOunces(1)).to.equal(35);
+  });
 
+  it('should find the water intake for a user on a specified date', () => {
+    expect(hydration.calculateDailyOunces(1, "2019/06/15")).to.equal(37);
+    expect(hydration.calculateDailyOunces(2, "2018/10/24")).to.equal(40);
+  });
+
+  //these last two functions aren't working properly
+  it('should find water intake by day for first week', () => {
+    expect(hydration.calculateFirstWeekOunces(userRepo, 1)).to.eql(['2019/06/15: 37', '2018/06/16: 39', '2016/08/22: 30']);
+    //this function needs the date as a parameter
+  });
```

Comment:
I agree this is kind of a funky function and should be worked on for UI, if you trace back the function "getFirstWeek" it contains another function "makesortedUserArray"     and slicing the first 7 indexes in the array.  makeSorted sorts the original array by date .  this definately has some holes in the logic and calculateFirstWeek should probably be renamed to make it reflect what it is actually returning